modules/apps/portal/portal-default-permissions-web/src/main/java/com/liferay/portal/defaultpermissions/web/internal/configuration/PortalDefaultPermissionsGroupConfigurationImpl.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.defaultpermissions.web.internal.configuration;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+import com.liferay.portal.configuration.module.configuration.ConfigurationProvider;
+import com.liferay.portal.defaultpermissions.configuration.PortalDefaultPermissionsConfiguration;
+import com.liferay.portal.defaultpermissions.configuration.PortalDefaultPermissionsGroupConfiguration;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.util.HashMapDictionaryBuilder;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Stefano Motta
+ */
+@Component(
+	property = "portal.default.permissions.scope=group",
+	service = PortalDefaultPermissionsConfiguration.class
+)
+public class PortalDefaultPermissionsGroupConfigurationImpl
+	implements PortalDefaultPermissionsConfiguration {
+
+	@Override
+	public Map<String, Map<String, String[]>> getDefaultPermissions(
+		long companyId, long groupId) {
+
+		try {
+			PortalDefaultPermissionsGroupConfiguration
+				portalDefaultPermissionsGroupConfiguration =
+					_configurationProvider.getGroupConfiguration(
+						PortalDefaultPermissionsGroupConfiguration.class,
+						groupId);
+
+			String defaultPermissions =
+				portalDefaultPermissionsGroupConfiguration.defaultPermissions();
+
+			if (Validator.isNull(defaultPermissions)) {
+				return new HashMap<>();
+			}
+
+			return _objectMapper.readValue(defaultPermissions, _typeReference);
+		}
+		catch (Exception exception) {
+			_log.error(exception);
+		}
+
+		return null;
```

Comment:
and here it's null?

Reply:
In this case I'm checking for the specific model; in this case returns null because I have to identify when it doesn't exists or is an empty map. If it doesn't exist I can check in the "company" settings for the same model